.gitignore

```diff
@@ -2,3 +2,5 @@
 /bower_components
 /manifest.json
 /node_modules
+.idea/
+*.swp
```

Comment:
I would recommend adding these to your global `.gitignore`. That way they don't have to be added to every `.gitignore` in every project you work on.